src/ui/src/app/modules/user/pages/game-view/game-view.component.ts

```diff
@@ -60,7 +60,7 @@ export class GameViewComponent implements OnInit, AfterViewInit, OnDestroy {
     this.newGameSubscription$?.unsubscribe();
     this.routerSubscription$?.unsubscribe();
     this.gameProgress?.unsubscribe();
-    // TODO: Analyse if memory leak problem could arise
+    //TODO: Analyse if memory leak problem could arise
```

Comment:
Is this TODO done? Does it still need to be done?

Reply:
honestly don't know can check it in my next PR tho

Reply:
ok